src/app/Shared/Redux/Configurations/DashboardConfigSlice.tsx

```diff
@@ -261,6 +266,14 @@ const getTemplateHistoryIndexForMutation = (state: DashboardConfigState, templat
 
 export const dashboardConfigReducer = createReducer(INITIAL_STATE, (builder) => {
   builder
+    .addCase(dashboardConfigClearAllCardsIntent, (state) => {
+      const currentLayout = state.layouts[state.current];
+      if (currentLayout.cards.length < 1) {
+        state.errorMessage = 'No cards to clear.';
```

Comment:
> @tthvo please check the new push. I am trying to display error message if a user clears empty cards. its not happening though!

@aali309 The error view is not needed here tho. If the layout is empty, the `clear all` action can be either:
- No-op (no effect).
- The `Clear All Cards` button can be disabled (I recommend this so that UI does not appear frozen).